src/server/worker/workerrequests.js

```diff
@@ -16,9 +16,9 @@ var Core = requireJS('common/core/coreQ'),
     constraint = requireJS('common/core/users/constraintchecker'),
```

Comment:
Out of curiosity are all of the additions consequences of the refactor or fixes/improvements?